app/domain/authentication/authenticator_input.rb

```diff
@@ -9,7 +9,7 @@ class AuthenticatorInput < ::Dry::Struct
     attribute :service_id, ::Types::NonEmptyString.optional
     attribute :account, ::Types::NonEmptyString
     attribute :username, ::Types::NonEmptyString.optional
-    attribute :request_body, ::Types::String.optional
+    attribute :credentials, ::Types::String.optional
```

Comment:
let's feel aligning here for this collection of attributes

Reply:
what? if you mean to align the `::Types` then i prefer not to as it is not configurable.